spec/std/int_spec.cr

```diff
@@ -537,6 +537,18 @@ describe "Int" do
     end
   end
 
+  {% if compare_versions(Crystal::VERSION, "2.6.1") > 0 %}
```

Comment:
@asterite is this version correct? Shouldn't be `0.26.1` instead?

Reply:
Oops! Fixed :-)